src/AdminConsole/ApiHelper.cs

```diff
@@ -24,6 +24,16 @@
~
~public class PasswordlessManagementOptions
 {
     public string ApiUrl { get; set; }
     public string ManagementKey { get; set; }
+
+    private string? _publicApiUrl;
+    public string PublicApiUrl

```

Comment:
I don't think this is needed. We should probably use the "ApiUrl" of the PasswordlessOptions instead, as this configuration is just for the endpoints where ManagementKey is used